java/src/main/native/ai_onnxruntime_OrtSession.c

```diff
@@ -13,227 +13,286 @@
  * Method:    createSession
  * Signature: (JJLjava/lang/String;J)J
  */
-JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OrtSession_createSession__JJLjava_lang_String_2J
-  (JNIEnv * jniEnv, jclass jclazz, jlong apiHandle, jlong envHandle, jstring modelPath, jlong optsHandle) {
-    (void) jclazz; // Required JNI parameter not needed by functions which don't need to access their host object.
-    const OrtApi* api = (const OrtApi*) apiHandle;
-    OrtSession* session;
+JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OrtSession_createSession__JJLjava_lang_String_2J(JNIEnv* jniEnv, jclass jclazz, jlong apiHandle, jlong envHandle, jstring modelPath, jlong optsHandle) {
+  (void)jclazz;  // Required JNI parameter not needed by functions which don't need to access their host object.
+  const OrtApi* api = (const OrtApi*)apiHandle;
+  OrtSession* session = NULL;
```

Comment:
>rtSession* session = NULL; [](http://example.com/codeflow?start=3&length=26)

Can we move this declaration just before it is used?

Reply:
I'd have to duplicate it so it was in both branches of the ifdef then, and that seemed unnecessary.